CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# 2.0.0 (2017-06-25) (unpublished)
```

Comment:
Manually adding the changes from the previous PR that weren't published b/c of the bug this PR fixes.